.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v8
+        with:
+          exempt-all-pr-assignees: true
+          exempt-pr-labels: 'pinned'
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

Comment:
```suggestion
          stale-issue-message: 'This issue is stale because it has been open for 30 days with no activity. Remove stale label or add a comment, otherwise it will be closed in 5 days.'
```